### DISCUSSION TITLE
33 Conceptos que todo desarrollador de JavaScript debería saber

### DISCUSSION THREAD

**Discussion by osorniochris** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#356] [Ideas]
Hola explorers, les dejo este repo de referencia sobre conceptos de js. Es mucha teoría, pero si a alguien le interesa profundizar y comprender a fondo el lenguaje le puede ser útil :)

[33 Concepts Every JavaScript Developer Should Know](https://github.com/leonardomso/33-js-concepts)

Este canal de Youtube explica algunos de ellos (y en español), por si se les hace más fácil.
[La Cocina del Código](https://www.youtube.com/c/LaCocinadelC%C3%B3digo)

**Comment by Dfredude** [comment]
Que recurso tan chingon! Gracias

**Comment by JuanSalvador73** [comment]
Mil gracias por el aporte :D
